test/index.ts

```diff
@@ -39,19 +42,23 @@ const p = proxyquire('../src', {
 
 const sandbox = sinon.createSandbox();
 
+// eslint-disable-next-line no-undef
```

Comment:
To fix this one, make sure `afterEach` is imported as part of the mocha requires.

```js
import {describe, it, afterEach} from 'mocha';
```

Then you should be able to get rid of these exceptions

Reply:
thanks